src/nft/pages/asset/Asset.tsx

```diff
@@ -37,6 +50,10 @@ const Asset = () => {
     [data, gqlData, isNftGraphQl]
   )
 
+  const { data: collectionStats } = useQuery(['collectionStats', contractAddress], () =>
```

Comment:
Are there currently any fields missing from the collection object returned from GraphQL that collectionStats has? If not can remove this extra input and query?

Reply:
the social links are not returned in the original query.  Not sure if that will change with graphql

Reply:
it can be added to the gql query, can sync on that add after this PR.